inst/rmarkdown/homepage/index.Rmd

```diff
@@ -0,0 +1,363 @@
+---
+title: INBO protocols
+date: '`r Sys.Date()`'
+---
+
+# Welkom! / Welcome!
+
+Voor Nederlands [klik hier](#welkom).
+
+For English [click here](#welcome).
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = FALSE,
+  eval = TRUE,
+  warning = FALSE,
+  error = TRUE,
+  message = FALSE
+)
+library(htmlwidgets)
+library(slickR)
+library(reactable)
+```
+
+
+```{r}
+slickR(obj = c("voorbeeld1.png", "voorbeeld2.jpeg", "voorbeeld3.png",
+               "voorbeeld4.jpg", "voorbeeld5.png", "voorbeeld6.jpg"),
+       width = "95%") +
+  settings(dots = TRUE, adaptiveHeight = TRUE,
+           autoplay = TRUE)
+```
+
+
+# Nederlandstalige richtlijnen
+
+### Welkom!
+
+Protocollen verhogen de kwaliteit van veldwerk en labowerk en daarmee de kwaliteit van het wetenschappelijk onderzoek, dat transparant, traceerbaar en toepasbaar moet zijn.
+Ze dragen ertoe bij dat veldwerk beter geharmoniseerd is en beter herhaalbaar, reproduceerbaar en vergelijkbaar wordt. Ook vanuit efficiëntie oogpunt is een standaardisatie interessant voor het Instituut voor Natuur- en Bosonderzoek (INBO).
+
+Deze site doet dienst als een repository voor de meest recente versies van goedgekeurde, publiek raadpleegbare protocollen die op het INBO gebruikt worden.
+Elk protocol kan ofwel online via deze website geraadpleegd worden (html versie) of er kan een PDF versie gedownload worden.
+Aan de linkerkant vind je de navigatie naar de verschillende rubrieken.
+In de NEWS sectie vind je welke (versies van) protocollen wanneer werden toegevoegd.
+
+Deze website bevat de gepubliceerde protocollen.
+Het beheer, onderhoud, goedkeuring en revisie van de bronbestanden die aan de basis liggen van deze gepubliceerde protocollen gebeurt in de [inbo/protocolsource](https://github.com/inbo/protocolsource) GitHub repository.
+
+### Vijf soorten protocollen
+
+We onderscheiden vijf soorten standaardprotocollen.
+Het standaard veldprotocol (`sfp`) wordt gebruikt om de stappen uit te leggen die een veldwerker in het veld nodig heeft om een specifieke veldactiviteit uit te voeren.
+Het tweede type is het standaard instrumentprotocol (`sip`) dat het gebruik en het onderhoud van toestellen uitlegt.
+Het derde type zijn de standaard analytische protocollen (`sap`) die in het laboratorium of in het veld worden gebruikt om een bepaalde laboratorium- of veldanalytische procedure uit te leggen waarbij fysische of chemische metingen worden verricht.
+Het vierde type staat voor standaard operationele procedures (`sop`) en is voor alles wat niet in een van de vorige types past.
+
+Het vijfde type is een categorie die gereserveerd is voor projectspecifieke protocollen (`spp`).
```

Comment:
Ik vraag me ook af waarom een deel in het enkelvoud is, en een deel in het meervoud.  De vorige heb ik naar enkelvoud gebracht, voor deze past dit niet in de zin, vandaar dat ik het meervoud behouden heb.  Aan u om te beslissen wat je ermee doet: alles in enkelvoud (waarvoor je de zin best aanpast), of deze ene in meervoud laten.
```suggestion
Het vijfde type is een categorie die gereserveerd is voor **projectspecifieke protocollen** (`spp`).
```